sispmctl.coffee

```diff
@@ -38,37 +38,49 @@ module.exports = (env) ->
       @name = config.name
       @id = config.id
       super()
+      @_updateStatus() if @config.interval > 0
 
-    getState: () ->
-      if @_state? then return Promise.resolve @_state
-      # Built the sispmctrl command to get the outlet status
+    _updateStatus: () ->
+      setTimeout () =>
+        @_getState()
+        .finally () =>
```

Comment:
We should handle errors (exception) here by logging them and then calling done() on the promise chain.